kars/NNNN-kar-template/README.md

```diff
@@ -0,0 +1,146 @@
+<!--
+**Note:** When your KAR is complete, all of these comment blocks should be removed.
+
+Follow the guidelines of the [documentation style guide].
+In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+
+To get started with this template:
+
+- [ ] **Create an issue in kubernetes-sigs/wg-ai-conformance**
+  When filing an AI conformance requirement tracking issue, please make sure to complete all
+  fields in that template. One of the fields asks for a link to the KAR. You
+  can leave that blank until this KAR is filed, and then go back to the
+  issue and add the link.
+- [ ] **Make a copy of this template directory.**
+  Copy this template into the owning directory and name it
+  `NNNN-short-descriptive-title`, where `NNNN` is the issue number (with no
+  leading-zero padding) assigned to your AI conformance requirement issue above.
+- [ ] **Fill out as much of the kar.yaml file as you can.**
+  At minimum, you should fill in the "Title", "Authors", "Status", and date-related fields.
+- [ ] **Fill out this file as best you can.**
+  At minimum, you should fill in the "Description" sections.
+- [ ] **Create a PR for this KAR.**
+  Assign it to wg-ai-conformance leads who are sponsoring this process.
+- [ ] **Merge early and iterate.**
+  Avoid getting hung up on specific details and instead aim to get the goals of
+  the KAR clarified and merged quickly. The best way to do this is to 
+  start with the high-level sections and fill out details incrementally in
+  subsequent PRs.
+
+Just because a KAR is merged does not mean it is complete or approved. Any KAR
+marked as `provisional` is a working document and subject to change. You can
+denote sections that are under active debate as follows:
+
+```
+<<[UNRESOLVED optional short context or usernames ]>>
+Stuff that is being argued.
+<<[/UNRESOLVED]>>
+```
+
+When editing KARS, aim for tightly-scoped, single-topic PRs to keep discussions
+focused. If you disagree with what is already in a document, open a new PR
+with suggested changes.
+
+One KAR corresponds to one "AI conformance requirement" for its whole lifecycle.
+You do not need a new KAR to move from SHOULD to MUST, for example. If
+new details emerge that belong in the KAR, edit the KAR. Once a requirement has become
+"implemented", major changes should get new KARs.
+
+The canonical place for the latest set of instructions (and the likely source
+of this file) is [here](/kars/NNNN-kar-template/README.md).
+
+**Note:** Any PRs to move a KAR to `implementable`, or significant changes once
+it is marked `implementable`, must be approved by each of the KAR approvers.
```

Comment:
Same question here. Perhaps we need to define what "implement" means for AI conformance. Does it mean that the requirement is merged in the CNCF repo, test is implemented, or something else?